src/components/nav/NavLink.js

```diff
@@ -88,9 +111,20 @@ NavLink.propTypes = {
   href: PropTypes.string,
 
   /**
-   * Apply 'active' style to this component
+   * Apply 'active' style to this component. Set to "exact" to automatically
+   * toggle active status when pathname matches href, or to "partial" to
+   * automatically toggle on a partial match.
+   *
+   * For example
+   * - dbc.NavLink(..., href="/my-page", active="exact") will be active on
+   *   "/my-page" but not "/my-page/other" or "/random"
+   * - dbc.NavLink(..., href="/my-page", active="partial") will be active on
+   *   "/my-page" and "/my-page/other" but not "/random"
    */
```

Comment:
Is is worth having an explanation of the format an href must take? From what I can see this doesn't account for someone using `http://...`

Reply:
Yes, this is a good call. I'll try to clarify the docstring / documentation.